### PR TITLE
fix --modules-path in nginx package

### PR DESCRIPTION
**Pull request by lpcalisi**
This PR fix:

`modules-path` was setted by default on */var/lib/nginx* while module subpackages install modules over */usr/local/nginx/modules*.

When you to add module have this error:

image

`--modules-path` is not taking [$_modules_dir](https://github.com/wolfi-dev/os/blob/main/nginx.yaml#L57) var

image

### CODE REVIEW

**Comment by lpcalisi** [comment]
@joshrwolf could run again pipeline? I've rebuilt locally for x86_64 and aarch64, and it's works ok.

**Comment by joshrwolf** [comment]
thanks for catching this!

File: nginx.yaml [hunk 1]
```diff
@@ -138,8 +138,6 @@ subpackages:
           mkdir -p ${{targets.subpkgdir}}/usr/lib/nginx/modules
           mkdir -p ${{targets.subpkgdir}}/etc/nginx/modules
 
-          mv ${{targets.destdir}}/ngx_${{range.key}}_module.so ${{targets.subpkgdir}}/usr/lib/nginx/modules/
```
Comment (joshrwolf) [inline nginx.yaml:141]: does this `mv` need to be removed?

Comment (lpcalisi) [reply nginx.yaml:141]: yes, because in the compilation the modules are generated in `/usr/lib/nginx/modules`

Comment (lpcalisi) [reply nginx.yaml:141]: you are right josh, the mv changes but it's needed. I've push the change.

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**